Anime Shitai/dist/metadata.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "author": {
+    "name": "Priim0",
+    "id": "774995448485773342"
+  },
+  "service": "Anime Shitai",
+  "description": {
+    "en": "Anime Shitai is a German Anime Source where you can watch Animes here with german subtitle and german dubbed. We have over 1000 Animes that wait for you to watch!",
+    "de": "Anime Shitai ist eine Anime Stream Source wo ihr über 1000 kostenlose Anime Serien anschauen könnt, Jede woche gibt auch noch die neuste Anime Folgen frisch aus Japan mit deutschem Untertitel nur für euch und das völlig Kostenlos auf Full HD / 1080p ",
+    "fr": "Anime Shitai est un portail d'anime allemand où vous pouvez regarder des animes en allemand ou avec des sous-titres allemands. Nous avons plus de 1000 animes qui vous attendent!"
+  },
+	"url": [
+    "www.anime-shitai.com",
+    "anime-shitai.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/8PWKaRe.jpg",
```

Comment:
Is this the best logo you could find? It looks like one of those Fake PNGs you find on Google

Reply:
Its not a Fake PNG, its the logo from our website since years.

Reply:
Hmm okay, could probably get a better quality image for the sake of consistency if a better quality one exists.